### PR TITLE
ci: add markdown lint and fixed markdown files

### PR DESCRIPTION
**Pull request by Arbiv**
## Which problem is this PR solving?
#396

## Short description of the changes
Added lint for markdown

### CODE REVIEW

File: plugins/node/opentelemetry-instrumentation-fastify/src/version.ts [hunk 1]
```diff
@@ -1,18 +0,0 @@
-/*
```
Comment (dyladan) [inline plugins/node/opentelemetry-instrumentation-fastify/src/version.ts:1]: Good catch but this might be better done in a separate PR since it has nothing to do really with what the PR is about.

**Review by obecny — APPROVED** [review]
lgtm

**Comment by Arbiv** [comment]
UT failure and coverage decrease seems irrelevant to the changes in the PR

**Comment by Arbiv** [comment]
@dyladan I fixed the comment, can you please re-review it?

**Review by vmarchaud — APPROVED** [review]

**Comment by vmarchaud** [comment]
@Arbiv Could you please fix the conflict so we can merge this ?

**Comment by Arbiv** [comment]
@vmarchaud Fixed conflict
